applications/launchpad/gui-react/src/components/CoinsList/index.tsx

```diff
@@ -19,10 +19,7 @@ const CoinsList = ({ coins, color }: CoinsListProps) => {
   return (
     <StyledCoinsList color={color}>
       {coins.map((c, idx) => (
-        <CoinsListItem
-          key={`coin-${idx}`}
-          loading={c.loading ? c.loading.toString() : 'false'}
-        >
+        <CoinsListItem key={`coin-${idx}`} $loading={c.loading}>
```

Comment:
why add $ to the prop name, what is the significance of this character? it looks bizarre

Reply:
neat